fccodegen/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+Main module for Web Lab code generation
+"""
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+
+#
+# Version info: Remember to keep this in sync with setup.py!
```

Comment:
Or use [versioneer](https://pypi.org/project/versioneer/)

Reply:
Is this alright?

https://github.com/ModellingWebLab/fccodegen/blob/initial_setup/fccodegen/__init__.py#L26-L27
https://github.com/ModellingWebLab/fccodegen/blob/initial_setup/setup.py#L11-L13

Reply:
OK :)
Make sure to use it on the `version=` line though!

Reply:
woops!